client/state/ui/selectors/site-data.ts

```diff
@@ -52,5 +52,6 @@ export interface SiteDataOptions {
 	is_difm_lite_in_progress: boolean;
 	is_domain_only: boolean;
 	difm_lite_site_options?: DIFMLiteSiteOptions;
+	updated_at: string | undefined;
```

Comment:
This needs to be:

```suggestion
	updated_at?: string;
```

If we explicitly set `undefined` as part of a union type, we'll need to explicitly pass `undefined` to have the types working, meaning it's not optional. To make this property optional, we need to suffix it with a question mark instead.

Try it: https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgCoE8AOKDeBYAKAEg4AuZAIwHsqAbCOEZAH2QFcQATCGUCTwkQoB+ctToMQhAL6FCCKiADOYZFQoArchmzIAvMhzSgA

Check how the compiler does not complain about `b`, but does complain about `a`.

Reply:
Thanks for the example. I've changed that new property to be optional.